cfgov/v1/models/learn_page.py

```diff
@@ -39,6 +39,7 @@ class AbstractFilterPage(CFGOVPage):
 
 
     # Configuration tab panels
+    header_panel = [StreamFieldPanel('header')]
```

Comment:
I had to add that to get the standard Header panel from AbstractFilterPage into my subclass. See this line: https://github.com/cfpb/cfgov-refresh/pull/2321/files#diff-c53bb3e17622a41cfcf254dedc4920dcR135